ci/scripts/gitlab/report_test_results.py

```
@@ -199,14 +203,26 @@ def main():
-                                blocks=report_messages.failure_blocks,
-                                thread_ts=response["ts"])
+        failure_blocks = report_messages.failure_blocks or []
+        failure_text = report_messages.failure_text or []
+        if len(failure_blocks) > SLACK_BLOCK_LIMIT:
+            blocks_chunks = chunk_items(failure_blocks, SLACK_BLOCK_LIMIT)
+            text_chunks = chunk_items(failure_text, SLACK_BLOCK_LIMIT)
+        else:
+            blocks_chunks = [failure_blocks]
+            text_chunks = [failure_text]
+
+        for blocks_chunk, text_chunk in zip(blocks_chunks, text_chunks):
```

Ruff (0.14.14)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)